routemaster/db/model.py

```diff
@@ -50,8 +53,8 @@
     Column('forced', Boolean, default=False),
 
     # Null indicates starting a state machine
-    Column('old_state', String, nullable=True),
-    Column('new_state', String),
+    NullableColumn('old_state', String),
+    NullableColumn('new_state', String),
```

Comment:
Is this change intentional? (this column was previously non-nullable)

Reply:
Actually columns are nullable by default, and we even use the null value here when we delete a label.

Reply:
So they are. My question was more around the underling intention. As discussed IRL though we do depend on this being nullable; so good to have this clarified in the models.